Put testing examples for programming challenges into a separate markdown file (#1405)
uccser/cs-unplugged: csunplugged/topics/content/en/error-detection-and-correction/programming-challenges/check-valid-credit-card/testing-examples.md

## Testing examples:

Your program should display the outputs shown in this table for the given
inputs provided:

> *Important: None of these is an actual credit card number, but the correct

Reviewer: ```suggestion
*Important: None of these are actual credit card numbers, but the correct
```

Reviewer: Might as well do a grammar check too
